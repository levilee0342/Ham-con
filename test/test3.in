-1395
-82
-18668
1653
-26168
-2857
-26561
-2759
-26140
1419
