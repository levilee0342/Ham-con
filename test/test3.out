1653
-26561